tdrs-frontend/src/actions/sttList.test.js

```diff
@@ -85,5 +85,8 @@ describe('actions/stts.js', () => {
     const actions = store.getActions()
     expect(actions[0].type).toBe(FETCH_STTS)
     expect(actions[1].type).toBe(SET_STTS_ERROR)
+    expect(actions[1].payload).toStrictEqual({
```

Comment:
I thought this test could be a little bit more specific so I'm ensuring that an error actually gets passed.